crud-framework-core/src/main/java/dev/krud/crudframework/crud/handler/CrudDeleteHandlerImpl.java

```diff
@@ -75,4 +77,54 @@ public <ID extends Serializable, Entity extends BaseCrudEntity<ID>> void deleteI
 			postHook.run(entity);
 		}
 	}
+
+    @Override
+    public <ID extends Serializable, Entity extends BaseCrudEntity<ID>> void bulkDelete(List<ID> ids, Class<Entity> entityClazz, HooksDTO<CRUDPreDeleteHook<ID, Entity>, CRUDOnDeleteHook<ID, Entity>, CRUDPostDeleteHook<ID, Entity>> hooks, boolean applyPolicies) {
+
+        Class<ID> idClass = (Class<ID>) ids.get(0).getClass();
```

Comment:
I think we should start relying on reflection less and EntityMetadataDTO more, meaning fetch metadata here and add if neeeded its ID class and ID name

Reply:
Already being fetched in this method so would make sense to move it up and add the additional info